JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -163,10 +162,10 @@ private fun Paragraph(paragraph: Paragraph) {
 
     val annotatedString = paragraphToAnnotatedString(
         paragraph,
-        MaterialTheme.typography(),
-        MaterialTheme.colors().codeBlockBackground
+        MaterialTheme.typography,
```

Comment:
Either pass `typography` or inline that val throughout.

Reply:
Can you clarify, not sure what update your proposing.

Reply:
You hold `MaterialTheme.typography` as a local val on line 122 so it feels strange to grab it again here. I'd suggest either passing the local val into this method, or not holding a val.

Reply:
Ah ok, doing so thanks!

Reply:
Oic, that's a different function – I think it got confused due to github formatting.